### PR TITLE
Add example results for User Guide documents

### PR DESCRIPTION
**Pull request by Genius-pig**
according to https://github.com/apache/iotdb/issues/2079

### CODE REVIEW

**Review by Alima777 — CHANGES_REQUESTED** [review]
Hi,
1. Better to use the system command console instead of IDEA console to keep the input style.
2. To indicate the format of `align by device`, it's better to involve multi devices in result set.

**Comment by Genius-pig** [comment]
> Hi,
> 
> 1. Better to use the system command console instead of IDEA console to keep the input style.
> 2. To indicate the format of `align by device`, it's better to involve multi devices in result set.

I will use print information instead of photos.

File: docs/UserGuide/Operation Manual/DDL Data Definition Language.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ IoTDB > set storage group to root.sgcc
 
 We can thus create two storage groups using the above two SQL statements.
 
-It is worth noting that when the path itself or the parent/child layer of the path is already set as a storage group, the path is then not allowed to be set as a storage group. For example, it is not feasible to set `root.ln.wf01` as a storage group when two storage groups `root.ln` and `root.sgcc` exist. The system gives the corresponding error prompt as shown below:
+It is worth noting that when the path itself, or the parent/child layer of the path is already set as a storage group, the path is then not allowed to be set as a storage group. For example, it is not feasible to set `root.ln.wf01` as a storage group when two storage groups `root.ln` and `root.sgcc` exist. The system gives the corresponding error prompt as shown below:
```
Comment (jixuan1989) [inline docs/UserGuide/Operation Manual/DDL Data Definition Language.md:35]: no ","

File: docs/UserGuide/Operation Manual/DML Data Manipulation Language.md [hunk 2]
```diff
@@ -752,6 +756,11 @@ The SQL statement is:
 select s1,s2 from root.sg1.* align by device
 ```
 
+The result shows below:
+
+<center><img style="width:100%; max-width:800px; max-height:600px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/26118649/100957517-94362200-3555-11eb-932c-511ea629f317.png"></center>
```
Comment (jixuan1989) [inline docs/UserGuide/Operation Manual/DML Data Manipulation Language.md:761]: why not change all these figures as tables..

File: docs/UserGuide/Operation Manual/DDL Data Definition Language.md [hunk 3]
```diff
@@ -41,15 +41,25 @@ Msg: org.apache.iotdb.exception.MetadataException: org.apache.iotdb.exception.Me
...
+```
++-------------+
+|storage group|
++-------------+
+|    root.sgcc|
+|      root.ln|
++-------------+
+Total line number = 2
+It costs 0.060s
+```
```
Comment (HTHou) [inline docs/UserGuide/Operation Manual/DDL Data Definition Language.md:62]: Add result of `show storage group root.ln` like previous graph?

File: docs/UserGuide/Operation Manual/DML Data Manipulation Language.md [hunk 4]
```diff
@@ -194,9 +273,15 @@ select count(status) from root.ln.*.* group by level=2
...
++----------------------------+----------------------------+
+|COUNT(root.ln.wf01.*.status)|COUNT(root.ln.wf02.*.status)|
++----------------------------+----------------------------+
+|                       10080|                       10082|
++----------------------------+----------------------------+
+Total line number = 1
+It costs 0.003s
+```
 
 All supported aggregation functions are: count, sum, avg, last_value, first_value, min_time, max_time, min_value, max_value.
```
Comment (HTHou) [inline docs/UserGuide/Operation Manual/DML Data Manipulation Language.md:286]: How about add more example results like sum or last_value?

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**